src/mutator.ts

```diff
@@ -16,20 +16,17 @@ export default function mutator<T extends ActionMessage>(
         throw new Error('Mutators can only subscribe to action creators.');
     }
 
-    // Wrap the callback in a MobX action so it can modify the store
-    let wrappedTarget = action(target);
-
     // Subscribe to the action
-    subscribe(actionId, (actionMessage: T) => {
+    subscribe(actionId, action((actionMessage: T) => {
```

Comment:
>action((actionMessage: T) => { [](start = 24, length = 30)

Nitpicky, but maybe you can pull this whole delegate out and assign it to `wrappedTarget` above.  And then we can preserve the comment about wrapping the callback above.